_posts/2016/2016-05-13-geecon.md

```diff
@@ -0,0 +1,86 @@
+- the compact and fluent syntax of lambdas can hide issues where the separation of concerns is violated
+
+I liked the talk because the topic was specific and small so it was possible to dive rather deep. Also they presented it in a really nice way by illustrating their points with live-coding and telling the whole story as a dialogue between two developers.
+
+> <blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr"><a href="https://twitter.com/DanielSawano">@DanielSawano</a> <a href="https://twitter.com/DanielDeogun">@DanielDeogun</a> great talk about pitfalls and best-practices with Java8 Lambdas - and fun too 💯 <a href="https://twitter.com/hashtag/geecon?src=hash">#geecon</a> <a href="https://t.co/gWpkvIozLK">pic.twitter.com/gWpkvIozLK</a></p>&mdash; Mathias D (@zaddo) <a href="https://twitter.com/zaddo/status/730666445688979456">12. Mai 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+## Julian Warszawski - "Greenfield Effect: Patterns for Effective Disaster Delivery"
+
+Julian Warszawski ([@hundredmondays](https://twitter.com/hundredmondays)) gave a very refreshing talk about the current state of our industry. He asked why developers so often start a greenfield project with best intentions and find themselves with a big ball of mud after some time that is harder and harder to maintain and delivers features slower and slower.
+
+One of the reasons he gave was technical debt and decisions taken by the wrong people. He stated that technical debt is a misleading term because the association to financial debt. Financial debt can easily be quantified and paid back in the future which is not the case for technical debt. And we better try to not pile up any because experience shows that it lasts forever and slows our work down.
+
+> <blockquote class="twitter-tweet" data-lang="de"><p lang="en" dir="ltr">just push this master. do <a href="https://twitter.com/hashtag/refactoring?src=hash">#refactoring</a> next week. <a href="https://t.co/6CfaNsV9gP">pic.twitter.com/6CfaNsV9gP</a></p>&mdash; Julian Warszawski (@hundredmondays) <a href="https://twitter.com/hundredmondays/status/714920417215918084">29. März 2016</a></blockquote>
+  <script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+He also described the *Dunning Kruger Effect* (not so skilled people tend to overestimate their skills) and the *Halo Effect* (we tend to listen to people that are attractive, famous or respected by others and give them credibility without taken their real skills into consideration) and tried to associate them to the world of software development. He argued that these effects lead to wrong decisions in projects. Julian proposed to rather rely on scientific methods - gather data for a question, test the hypothesis, and use the results for a conclusion.
```

Comment:
we tend to listen to people that are attractive, famous, or respected by others, and give them credibility without taken their real skills into consideration (add commas)